simple-mpc-mode.el

```diff
@@ -27,6 +27,8 @@
 
 ;;; Code:
 
+(declare-function simple-mpc "simple-mpc")
```

Comment:
Why is this needed?

Reply:
Ah, yes, because further down below (around line 54) is this line:

```lisp
(set (make-local-variable 'revert-buffer-function) #'simple-mpc)
```

and `#'simple-mpc` is not known to be defined without that the `declare-function`. And it can't be fixed with a `require` due to a cyclic dependency on `simple-mpc-mode` in `simple-mpc.el`